wiki/Tournaments/en.md

```diff
@@ -12,49 +12,50 @@ Official tournaments hosted by the [Tournament Management](https://osu.ppy.sh/g/
 
 osu! World Cup is a country-based 4v4 Team VS tournament following single-elimination rule. It is an international event and is usually done yearly with the osu!communities around the world working together to make a tournament to create a competitive nature.
 
-| Name                                  | Start      | End        | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
-|---------------------------------------|------------|------------|----------------------------------------------------|------------------------------------------------------|------------------------------------------------------|
-| [osu! World Cup #1](/wiki/Tournaments/OWC/1)     | 31/01/2011 | 08/04/2011 | ![][flag_TW] Taiwan                    | ![][flag_NL] Netherlands                 | ![][flag_KR] South Korea                 |
-| [osu! World Cup #2](/wiki/Tournaments/OWC/2)     | 28/10/2011 | 24/03/2012 | ![][flag_KR] South Korea               | ![][flag_JP] Japan                       | ![][flag_BR] Brazil                      |
-| [osu! World Cup #3](/wiki/Tournaments/OWC/3)     | 25/10/2012 | 20/01/2013 | ![][flag_KR] South Korea               | ![][flag_JP] Japan                       | ![][flag_TW] Taiwan                      |
-| [osu! World Cup 2013](/wiki/Tournaments/OWC/2013) | 15/10/2013 | 08/12/2013 | ![][flag_KR] South Korea               | ![][flag_TW] Taiwan                      | ![][flag_PL] Poland                      |
-| [osu! World Cup 2014](/wiki/Tournaments/OWC/2014) | 02/10/2014 | 14/12/2014 | ![][flag_JP] Japan                     | ![][flag_PL] Poland                      | ![][flag_DE] Germany                     |
-| [osu! World Cup 2015](/wiki/Tournaments/OWC/2015) | 01/10/2015 | 13/12/2015 | ![][flag_US] USA                       | ![][flag_CN] China                       | ![][flag_PL] Poland                      |
-| [osu! World Cup 2016](/wiki/Tournaments/OWC/2016) | 02/10/2016 | 18/12/2016 | ![][flag_US] USA                       | ![][flag_CN] China                       | ![][flag_KR] South Korea                     |
+| Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
+|------|-------|-----|----------------------------------------------------|------------------------------------------------------|------------------------------------------------------|
+| [osu! World Cup #1](/wiki/Tournaments/OWC/1) | 31. January 2011 | 08. April 2011 | ![][flag_TW] Taiwan | ![][flag_NL] Netherlands | ![][flag_KR] South Korea |
```

Comment:
is this really the decided style for dates? what is the full stop abbreviating?

Reply:
Yes, `DD. MMMM YYYY` is the format that was decided (by Loctav). This was discussed back in May of this year in the osu!dev Discord.

Reply:
jesus, we're going to need to do a wiki-wide replacement of this into something more correct.

Reply:
(i could've sworn i was around for that conversation and explicitly said to not use this format because it is not only wrong but *wrong*)

Reply:
I had thought so, since you had once told Loctav, in a pull request, to remove them (and so he did). What should the date format be then?

Reply:
without full stops is a good start. going forward we may want to consider something more standardised so we can show region-specific datetime client-side.